homeassistant/components/awair/config_flow.py

```diff
@@ -17,7 +18,7 @@ class AwairFlowHandler(ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
-    async def async_step_user(self, user_input: dict | None = None):
+    async def async_step_user(self, user_input: dict | None = None) -> FlowResult:
```

Comment:
I'm sure that the type hint for user_input could also be improved, but there are so many possibilities for `async_step_***` that I'm not sure it would be useful to implement via the pylint plugin.
I think just enforcing the FlowResult is already an improvement...